README.md

```diff
@@ -53,8 +53,8 @@ To ease migration from FindBugs to SpotBugs these three configuration properties
 
 ## Installation
 
-- clone this repository and build it: `gradle distZip` or download [distribution](https://github.com/TouK/sputnik/releases/download/sputnik-1.5.1/sputnik-1.5.1.zip)
-- copy distribution file `build/distributions/sputnik-1.5.1.zip` to your installation dir, e.g. `/opt/sputnik` and unzip it
+- clone this repository and build it: `gradle distZip` or download [distribution](https://repo1.maven.org/maven2/pl/touk/sputnik/2.2.3/sputnik-2.2.3.zip)
```

Comment:
Maybe it's better to link at [latest](https://github.com/TouK/sputnik/releases/latest) release on GitHub instead of hard link on Maven?

Reply:
If it is on Github releases, then I do not mind. I create this suggestion because the previous build was not there :)